Framework/Core/include/Framework/AnalysisTask.h

```diff
@@ -151,12 +151,16 @@ struct OutputObj {
     static_assert(std::is_base_of_v<TNamed, T>, "You need a TNamed derived class to use OutputObj");
     header::DataDescription desc{};
     if (object == nullptr) {
-      strncpy(desc.str, "__OUTPUTOBJECT__", 16);
+      std::string temps{label};
```

Comment:
No way. ;-) Use a char* and copy stuff into it.

Reply:
Good point, in fact if we are not using the title, we do not need the if(nullptr) check since the label is always set and is required to be unique. I've rewritten the whole thing with memset+memcpy.